### ISSUE TITLE
Need Exception Handling for Coding Mentor

### ISSUE THREAD

**Issue by PCain02** [GatorEducator/execexam#3]
Description: When using the Coding Mentor the program crashes when it is not given an expected model or url.

Expected Behavior: The code should not crash. Instead, it should notify the user of the unexpected input and offer an alternative.

Actual Behavior: The code crashes and gives a large error output. This is the end of the output.
```
BadRequestError: litellm.BadRequestError: LLM Provider NOT provided. Pass in the LLM provider you are trying to call. You passed model=https://docs.litellm.ai/docs/providers
 Pass model as E.g. For 'Huggingface' inference endpoints pass in `completion(model='huggingface/starcoder',..)` Learn more: https://docs.litellm.ai/docs/providers
 ```

Proposed Solution:  Implement expectation handling statements within the coding mentor that provide more useful feedback.

**Comment by gkapfham** [comment]
Hi @dyga01 and @hemanialaparthi and @PCain02 can you give a status update concerning the resolution of this issue?

**Comment by dyga01** [comment]
@gkapfham This issue has been resolved in PR #22
